packages/cmrjs/package.json

```diff
@@ -6,9 +6,13 @@
     "node": ">=12.18.0"
   },
   "scripts": {
-    "test": "../../node_modules/.bin/ava",
+    "clean": "rm -f ./*.js ./*.d.ts",
+    "debug": "NODE_ENV=test node --inspect-brk node_modules/ava/profile.js --serial tests/*.js",
```

Comment:
I realize you just re-ordered these, but this isn't right:  ava no longer includes the profiler, and ava isn't in the dev-dependencies at this level.